### PR TITLE
Delegation program urgent updates

### PR DESCRIPTION
**Pull request by qwadratic**

### CODE REVIEW

**Comment by qwadratic** [comment]
@barriebyron It seem to work locally but deployment pipeline failed.

File: docs/node-operators/foundation-delegation-program.mdx [hunk 1]
```diff
@@ -132,44 +132,73 @@ See the following uptime leaderboards for the latest uptime performance scores:
 
 ### What happens after I receive a delegation?
 
-#### Payout Addresses
+#### Payout Procedure
 
-You must send computed rewards as described below to the same address that was delegated to you.
+As per the Mina Foundation Delegation policy, there is a mandatory requirement on delegatees to use the latest version of [thew payout script](https://github.com/jrwashburn/mina-pool-payout) to manage the return of rewards.
```
Comment (barriebyron) [inline docs/node-operators/foundation-delegation-program.mdx:137]:
```suggestion
As per the Mina Foundation Delegation policy, there is a mandatory requirement on delegatees to use the latest version of the [payout script](https://github.com/jrwashburn/mina-pool-payout) to manage the return of rewards.
```

File: docs/node-operators/foundation-delegation-program.mdx [hunk 2]
```diff
@@ -132,44 +132,73 @@ See the following uptime leaderboards for the latest uptime performance scores:
...
+#### Payout Procedure
 
-You must send computed rewards as described below to the same address that was delegated to you.
+As per the Mina Foundation Delegation policy, there is a mandatory requirement on delegatees to use the latest version of the [payout script](https://github.com/jrwashburn/mina-pool-payout) to manage the return of rewards.
 
-#### Payout Frequency
+Mina Foundation has collaborated on updating the well-known payout script (by Jonathan Washburn and Gareth Davies) to reflect and support its Delegation Program Policy requirements
 
-Rewards must be distributed at least once for a given epoch.  You can send rewards more frequently, but the preferred method is to send one payment in the amount of your obligation.  This is the easiest method to avoid confusion in tracking payments, and will reduce the likelihood you will be incorrectly flagged as delinquent.
+This payout script includes
```
Comment (barriebyron) [inline docs/node-operators/foundation-delegation-program.mdx:141]:
```suggestion
This payout script includes:
```

File: docs/node-operators/foundation-delegation-program.mdx [hunk 3]
```diff
@@ -132,44 +132,73 @@ See the following uptime leaderboards for the latest uptime performance scores:
...
+- Mapping of delegating wallet addresses with the correct return wallet address
+- Calculation of the Reward amounts and paybacks of the remaining rewards to the delegators.
+- Calculation of the supercharged rewards to be burned and sent to the burn address.
+- Specific memo which includes the md5 hash of Pool’s Public Key (MD5 hash)
 
-All the rewards for epoch N must be delivered (ie. accepted in a block, not just sent) no later than slot number 3,500 of the next epoch. This gives you about a week to sort out these payments.
+To see more, please check the  [https://github.com/jrwashburn/mina-pool-payout/blob/main/README.md](README) where you can review the script and supporting README
```
Comment (barriebyron) [inline docs/node-operators/foundation-delegation-program.mdx:147]:
```suggestion
For more details and to review the script, see the [README](https://github.com/jrwashburn/mina-pool-payout/blob/main/README.md) file in the `mina-pool-payout` repository.
```

**Review by barriebyron — APPROVED** [review]
@qwadratic your build errors were the link syntax, I corrected. 
Approving this PR, ping for review after you make the same changes in #770

see https://github.com/o1-labs/docs2/wiki/Markdown-Syntax#links for link syntax

**Pull request merged**